src/controllers/csi/provisioner/processmoduleconfig.go

```diff
@@ -57,14 +57,27 @@ func (provisioner *OneAgentProvisioner) readProcessModuleConfigCache(tenantUUID
 	var processModuleConfig processModuleConfigCache
 	processModuleConfigCache, err := provisioner.fs.Open(provisioner.path.AgentRuxitProcResponseCache(tenantUUID))
 	if err != nil {
+		provisioner.removeProcessModuleConfigCache(tenantUUID)
 		return nil, err
```

Comment:
errors should be returned wrapped in `errors.WithStack(...)` so better stackstrace can be logged